makefile

```diff
@@ -75,21 +79,23 @@ medgen-disease-extract.obo: x-Disease-or-Syndrome.obo x-Neoplastic-Process.obo
 medgen-disease-extract.json: medgen-disease-extract.obo
 	owltools $< -o -f json $@
 
-medgen-disease-extract.owl: medgen-disease-extract.obo
+output/medgen-disease-extract.owl: medgen-disease-extract.obo | output/
 	owltools $< -o $@
 
 # SSSOM ----------------------------------
-# todo: comemented out old pipeline: remove
-#medgen.obographs.json:
-#	robot convert -i medgen-disease-extract.owl -o $@
-#
-#medgen.sssom.tsv: medgen.obographs.json
-#	sssom parse medgen.obographs.json -I obographs-json -m config/medgen.sssom-metadata.yml -o $@
-sssom: umls-hpo.sssom.tsv
+sssom: umls-hpo.sssom.tsv sssom-validate
+
+sssom-validate: umls-hpo.sssom.tsv
 	sssom validate umls-hpo.sssom.tsv
 	sssom validate hpo-mesh.sssom.tsv
 
-umls-hpo.sssom.tsv hpo-mesh.sssom.tsv: ftp.ncbi.nlm.nih.gov/pub/medgen/MedGenIDMappings.txt
+output/medgen.obographs.json: output/medgen-disease-extract.owl | output/
+	robot convert -i $< -o $@
+
+output/medgen.sssom.tsv: output/medgen.obographs.json | output/
```

Comment:
## All other, superfluous changes
@twhetzel The essential change in this PR is basically [a one-liner](https://github.com/monarch-initiative/medgen/pull/27#r1606099147). But I wanna explain why there's a lot in the diff.

As I was working on the bug fix I was getting other errors, so I started to fix those as well. But then I realized that these errors are actually non-issues because they were happening when I was running `make all` locally. However, for this repo, the build runs off of `make minimal`, which has no errors now that I've fixed that bug. So, in any case, these fixes and optimizations I've made will be good if we ever go back to having a use for `make all`. I suppose I should have split those into a separate PR, but I'm just realizing that now, sorry. If you'd like I can still do that.

Reply:
Thanks for the explanation.

Reply:
Do all the `make` goals still work as expected?

Reply:
Yep! As mentioned in my meeting, `make minimal`, which is for the build, works. The latest release that's currently up was created by running `make minimal` from this branch.

As I was explaining in the meeting, I also fixed parts of `make all` before remembering that I didn't need to do that. `make all` has been broken for a while and will continue to be broken until it ever might be needed. Though I think I really ought to do #28 (quick change) to make this clearer / not cause undue worry.